nifi-nar-bundles/nifi-azure-bundle/nifi-azure-processors/src/test/java/org/apache/nifi/processors/azure/storage/AbstractAzureDataLakeStorageIT.java

```diff
@@ -53,6 +54,7 @@ protected String getDefaultEndpointSuffix() {
     protected void setUpCredentials() throws Exception {
         ADLSCredentialsService service = new ADLSCredentialsControllerService();
         runner.addControllerService("ADLSCredentials", service);
+        runner.setProperty(service, AzureStorageUtils.CREDENTIALS_TYPE, AzureStorageCredentialsType.ACCOUNT_KEY);
```

Comment:
What is the purpose of this change?

Reply:
It was a bug in the ADLS integration tests due to [NIFI-8278](https://issues.apache.org/jira/browse/NIFI-8278) but I just spotted it now.